lms/templates/courseware/static_tab.html

```diff
@@ -6,6 +6,11 @@
 <%static:css group='style-course-vendor'/>
 <%static:css group='style-course'/>
 </%block>
+<%block name="js_extra">
+
+<%include file="../discussion/_js_body_dependencies.html" />
```

Comment:
@cahrens points out that the courseware can show discussion modules, which is why it needs the discussion dependency. It definitely seems wrong to add a dependency on discussions to static tabs which will never show discussion modules. Note that it is also confusing to add the body dependency without the corresponding head dependency.

Looking into the body file, it basically just includes "mathjax_include.html". I suggest including that directly instead just for clarity.